flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamConfig.java

```diff
@@ -663,6 +664,14 @@ public boolean shouldSortInputs() {
 		return config.get(SORTED_INPUTS);
 	}
 
+	public void setGraphContainingLoops(boolean graphContainingLoops) {
+		config.setBoolean(GRAPH_CONTAINING_LOOPS, graphContainingLoops);
+	}
+
+	public boolean isGraphContainingLoops() {
```

Comment:
Just to clarify: this is intended for failing late after the user has forced UC and then rescales, right?

Reply:
No, this is needed to disable restoring channel state for iterative jobs (so they run normally without restoring or failing).